zapcore/entry.go

```diff
@@ -219,11 +219,9 @@ func (ce *CheckedEntry) Write(fields ...Field) {
 	for i := range ce.cores {
 		err = multierr.Append(err, ce.cores[i].Write(ce.Entry, fields))
 	}
-	if ce.ErrorOutput != nil {
-		if err != nil {
-			fmt.Fprintf(ce.ErrorOutput, "%v write error: %v\n", ce.Time, err)
-			ce.ErrorOutput.Sync()
-		}
+	if ce.ErrorOutput != nil && err != nil {
```

Comment:
```suggestion
	if err != nil && ce.ErrorOutput != nil {
```

Reply:
This may be a little clearer logically -- if there's an error, *then* we care about whether we have an `ErrorOutput`.

Reply:
thanks. I agree with you.